docs/conf/prague/2020/welcome-wagon.rst

```diff
@@ -58,6 +57,10 @@ Conference talks and Q&A
 - There’s a talk schedule on the **Reception** page and on the `Write the Docs website <https://www.writethedocs.org/conf/prague/2020/schedule/>`__. Select the Calendar icon next to the talks you want to see on the Reception page to add them to your calendar.
 - When it’s time for a talk you want to see, select **Stage**.
 - If we have problems with our online platform, go to the `Write the Docs website <https://www.writethedocs.org/conf/prague/2020/backup-livestream/>`__ for a back-up stream. This will only be turned on if we have technical issues.
```

Comment:
Good point, @mxsasha do you mind helping with this? We can just use some text that says "Here will be the live stream in case the conference platform blows up"...

Reply:
I've resolved this as discussed in https://github.com/writethedocs/www/pull/1345/commits/f12f085c2c0adda5464c837e2e08b0289cb9d2cd